services/src/main/java/org/apache/druid/cli/CliBroker.java

```diff
@@ -102,6 +103,12 @@ public CliBroker()
           binder.bindConstant().annotatedWith(Names.named("tlsServicePort")).to(8282);
           binder.bindConstant().annotatedWith(PruneLoadSpec.class).to(true);
           binder.bindConstant().annotatedWith(PruneLastCompactionState.class).to(true);
+          binder.bindConstant()
```

Comment:
It looks like this should work, but why not create a reasonable ResponseContextConfig and bind it directly to `ResponseContextConfig.class`? Seems like that would be simpler.

Reply:
Ah, there was only one constant to bind at first (`shouldFailOnTruncatedResponseContext`), and so I just added a constant binding. Later, to make the test coverage check happier, I added another constant `maxResponseContextHeaderSize` and just copy-pasted existing code 😅. Changed to directly bind the context class.